personio-personnel-data-api-oa3.yaml

```diff
@@ -1921,6 +1921,171 @@ paths:
+      - $ref: "#/components/parameters/XPersonioAppID"
+    get:
+      security:
+        - BearerAuth: [ ]
+      tags:
+        - Absences
+      description: This endpoint is responsible for fetching hourly absence data for the company employees.The result can be `paginated` and `filtered` by period and/or specific employee/employees. The result contains a list of hourly absence periods.
+      parameters:
+        - name: from
+          in: query
+          required: false
+          description: Filter by absence periods that start at or before given timestamp.
+          schema:
+            type: string
+            format: date-time
+        - name: to
+          in: query
+          required: false
+          description: Filter by absence periods that start at or before given timestamp.
+          schema:
+            type: string
+            format: date-time
+        - name: updated_from
+          in: query
+          required: false
+          description: Filter by periods that were created or modified from the date updated_from. It is inclusive, so all the periods created or modified from the beginning of the updated_from will be included in the results.
+          schema:
+            type: string
+            format: date-time
+        - name: updated_to
+          in: query
+          required: false
+          description: Filter by periods that were created or modified until the date updated_to. It is inclusive, so all the periods created or modified until the end of the updated_to will be included in the results.
+          schema:
+            type: string
+            format: date-time
+        - name: employees[]
+          in: query
+          required: false
+          description: A list of Personio employee ID's to filter the results. The result filters including only absences of provided employees
+          explode: true
+          schema:
+            type: array
+            items:
+              type: integer
+          allowReserved: true
+        - name: absence_types[]
+          in: query
+          required: false
+          description: A list of Personio absence type IDs to filter the results. The result filters including only absences of provided absence types.
+          explode: true
+          schema:
+            type: array
+            items:
+              type: string
+          allowReserved: true
+        - name: absence_periods[]
+          in: query
+          required: false
+          description: A list of Personio absence period IDs to filter the results. The result filters including only absences containing the provided ids.
+          explode: true
+          schema:
+            type: array
+            items:
+              type: string
+          allowReserved: true
+        - name: limit
+          in: query
+          required: false
+          description: Pagination attribute to limit how many attendances is returning per page
```

Comment:
attendances -> _absence periods_

Also, should probably rephrase. `how many absence periods is returning per page` doesn't really make sense.